native/cocos/renderer/gfx-vulkan/VKCommandBuffer.cpp

```diff
@@ -892,7 +892,7 @@ void CCVKCommandBuffer::pipelineBarrier(const GeneralBarrier *barrier, const Buf
                 bool fullBarrier = ccBarrier->getInfo().type == BarrierType::FULL;
                 bool missed = _barrierEvents.find(ccBuffer) != _barrierEvents.end();
                 if (!fullBarrier && !missed) {
-                    CC_ASSERT(_barrierEvents.find(ccBuffer) != _barrierEvents.end());
+                    // CC_ASSERT(_barrierEvents.find(ccBuffer) != _barrierEvents.end());
```

Comment:
It looks light the `CC_ASSERT` is needed. If `missed` is `false`, it means `fullBarrier` must be true to avoid the failing CC_ASSERT.

Reply:
ah, i got it, sorry about that. reverted